packages/components/src/components/radio-button/radio-button.ts

```diff
@@ -0,0 +1,236 @@
+import { css, html } from 'lit';
+import { customElement } from '../../../src/internal/register-custom-element';
+import { HasSlotController } from '../../internal/slot';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { property, query, state } from 'lit/decorators.js';
+import { watch } from '../../internal/watch';
+import componentStyles from '../../styles/component.styles';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary Radio buttons allow the user to select a single option from a group using a button-like control.
+ * @documentation https://solid.union-investment.com/[storybook-link]/radio-button
+ * @status stable
+ * @since 1.0
+ *
+ * @slot - The radio button's label.
+ * @slot icon - A presentational icon.
+ *
+ * @event sd-blur - Emitted when the button loses focus.
+ * @event sd-focus - Emitted when the button gains focus.
+ *
+ * @csspart base - The component's base wrapper.
```

Comment:
How do I set the border radius to 0 if I have multiple buttons? Did not get it to work...